src/main/kotlin/org/polyfrost/evergreenhud/hud/HeldItemLore.kt

```diff
@@ -0,0 +1,195 @@
+package org.polyfrost.evergreenhud.hud
+
+import cc.polyfrost.oneconfig.config.Config
+import cc.polyfrost.oneconfig.config.annotations.*
+import cc.polyfrost.oneconfig.config.annotations.Number
+import cc.polyfrost.oneconfig.config.core.OneColor
+import cc.polyfrost.oneconfig.config.data.Mod
+import cc.polyfrost.oneconfig.config.data.ModType
+import cc.polyfrost.oneconfig.hud.TextHud
+import cc.polyfrost.oneconfig.libs.universal.UGraphics
+import cc.polyfrost.oneconfig.libs.universal.UMinecraft
+import cc.polyfrost.oneconfig.renderer.NanoVGHelper
+import cc.polyfrost.oneconfig.renderer.TextRenderer
+import cc.polyfrost.oneconfig.utils.color.ColorUtils
+import cc.polyfrost.oneconfig.utils.dsl.mc
+import org.polyfrost.evergreenhud.mixins.GuiIngameAccessor
+import org.polyfrost.evergreenhud.utils.ItemStackUtils.getLore
+import kotlin.math.min
+
+/**
+    HeldItemLore.kt
+
+    A Kotlin class written by Erymanthus | RayDeeUx, Wyvest, and ImToggle
+    for the EvergreenHUD mod.
+
+    Displays the item lore of the currently held item (if there is any).
+
+    Allows for options to limit the number of lines to show, as well as skipping any
+    empty lore lines.
+
+    Also allows for omitting the item of the name from the HUD
+    (useful if VanillaHUD) is also installed.
+ */
+
+class HeldItemLore : Config(Mod("Held Item Lore", ModType.HUD), "evergreenhud/helditemlore.json", false) {
+    @HUD(name = "Main")
+    var hud = HeldItemLoreHud()
+
+    init {
+        initialize()
+    }
+
+    class HeldItemLoreHud : TextHud(false, 50, 50) {
+
+        @Switch(name = "Fade Out", description = "If disabled, the held item lore HUD will persist indefinitely.")
+        var fadeOut: Boolean = true
+
+        @Switch(name = "Instant Fade", description = "Requires \"Fade Out\" to be enabled, otherwise this toggle is ignored.")
```

Comment:
why not just `addDependency("instantFade", "fadeOut")` or whatever it is in kotlin if its different